idb/postgres/internal/writer/writer.go

```diff
@@ -104,6 +107,12 @@ var statements = map[string]string{
 		(addr, app, localstate, deleted, created_at, closed_at)
 		VALUES($1, $2, 'null'::jsonb, TRUE, $3, $3) ON CONFLICT (addr, app) DO UPDATE SET
 		localstate = EXCLUDED.localstate, deleted = TRUE, closed_at = EXCLUDED.closed_at`,
+	upsertAppBoxStmtName: `INSERT INTO app_box AS ab
+		(app, name, value, created_at)
+		VALUES ($1, $2, $3, $4)
+		ON CONFLICT (app, name) DO UPDATE SET
+		value = EXCLUDED.value`,
+	deleteAppBoxStmtName: `DELETE FROM app_box WHERE app = $1 and name = $2`,
```

Comment:
This is the only example I can see where we _actually_ delete a row, instead of marking it deleted. @winder - are you ok with this approach?

Reply:
Based on the earlier discussions about whether or not we care to keep historic boxes around, this is fine by me.